docs/index.rst

```diff
@@ -1,13 +1,13 @@
 .. _`Common Ground`: https://commonground.nl/
 
-
 PodiumD Autorisatie Beheer Component (PABC)
-=================
+===========================================
 
 .. toctree::
    :caption: ITA Documentatie
    :maxdepth: 2
    :hidden:
-
+   
    installation/index
    manual/index
+   api
```

Comment:
ik denk dat het netter is om dezelfde structuur te volgen als bij de rest van de documentatie. Dus hier verwijzen naar een api/index en dus en mapje 'api' met daarin een index bestand en een API-specificatie md of rst bestand